### DISCUSSION TITLE
Is the terminal in codespaces safe?

### DISCUSSION THREAD

**Discussion by D4RKH0R1Z0N** [community/community#17378] [Codespaces] [answered]
If a random person does `rm -rf /*` will it make the terminal Inaccessible for everyone?

**Comment by jshorty** [comment, marked as answer]
👋 Hi from Codespaces! Each codespace runs in a separate VM with its own attached resources, so the filesystem is yours to destroy if you like. Terminal activity in one codespace won't impact someone else's (unless you're sending network requests to one that's listening via a forwarded port, or similar).
